README.md

```diff
@@ -41,27 +41,29 @@ Run this command to install the SDK in your project:
 ```bash
 npm install --save @asyncapi/generator-model-sdk
 ```
+
 ## How it works
 
 The process of creating data models from input data consists of 2 processes, the input and generation process.
 
 ### The input process
 
 The input process ensures that any supported input is handled correctly, the basics are that any input needs to be converted into our internal model representation `CommonModel`. The following inputs are supported:
-- [JSON Schema Draft 7](#JSON-Schema-input), this is the default inferred input if we cannot find a another input processor.
+
+- [JSON Schema Draft 7](#JSON-Schema-input), this is the default inferred input if we cannot find an another input processor.
```

Comment:
Technically the `a` or `an` article is not needed. You are free to remove it.

Reply:
I use grammarly and sometimes it highlights such as these errors.